lighthouse-service/event_handler/evaluate_sli_handler.go

```diff
@@ -42,6 +42,12 @@ type criteriaObject struct {
 	CheckIncrease   bool
 }
 
+type KeySLI struct {
```

Comment:
do we need this struct somewhere else or would it make sense to make it unexported?

Reply:
it can be unexported, good catch!